lib/widgets/page.dart

```diff
@@ -265,6 +270,10 @@ class PageBodyEmptyContentPlaceholder extends StatelessWidget {
                   height: 23 / 17,
                 ).merge(weightVariableTextStyle(context, wght: 500)),
                 message!),
+              if (learnMoreButtonUrl != null)
+                ZulipWebUiKitButton(
+                  label: zulipLocalizations.learnMoreButtonLabel,
+                  onPressed: () => ZulipBinding.instance.launchUrl(learnMoreButtonUrl!)),
```

Comment:
I think we should use `PlatformActions.launchUrl` here to follow user's browser preference.